test/simplefootball_web_web/views/match_view_test.exs

```diff
@@ -0,0 +1,97 @@
+defmodule SimplefootballWebWeb.MatchViewTest do
+  use SimplefootballWebWeb.ConnCase, async: true
+
+  alias SimplefootballWeb.{Match, Team}
+  alias SimplefootballWebWeb.MatchView
+
+  test "render a match without teams" do
+    date = DateTime.truncate(DateTime.utc_now(), :second)
+
+    match = %Match{
+      date: date,
+      result: "2:1",
+      after_penalties: false,
+      extra_time: false
+    }
+
+    result = MatchView.render_match(match)
+
+    assert result == %{
+             date: DateTime.to_string(date),
+             result: "2:1",
+             afterPenalties: false,
+             extraTime: false
+           }
+  end
+
+  test "render a match with teams" do
+    date = DateTime.truncate(DateTime.utc_now(), :second)
+
+    match = %Match{
+      date: date,
+      result: "2:1",
```

Comment:
Bist du eigentlich mit diesem String format zufrieden? Ich finds herrlich pragmatisch und von mir aus kann es gerne so bleiben, aber ich sehe da schon einen alternativen Client mit regex drangehen, um es zu zerpflücken.

Reply:
Ich würde das erst einmal so lassen und wenn wir konkret den Fall haben, dann parsen wir das in zusätzliche Variablen auseinander auf dem Server